rplugin/python/nvim_ipy.py

```diff
@@ -265,7 +269,7 @@ def ipy_run(self, args):
             if p.get("source") == "page":
                 # TODO: if this is long, open separate window
                 self.append_outbuf(p['text'])
-
+        self.buf.name = self.base_buf_name + '({}s elapsed)'.format(time.time()-start_time) 
```

Comment:
I agree that it is pretty easy once you come up with the idea. It this also works nicely for visual mode:
````
let g:ipy_vprefix = "%%time \n"
vnoremap <Plug>(IPy-Run) :<c-u>call IPyRun(g:ipy_vprefix . <SID>get_visual_selection())<cr>
````
I think  I'd be nice to have such a feature out-of-the-box, as not everybody using the plugin should come up with their own a solution. The problem with this solution is that it is python specific.

>using kernel-side %time is more reliable, because if one enqueues several commands (i e >sends more requests before the last one is done) the timing will still be correct

Good point. The only reason to do it on the client side is that it would work for all kernels and the output could be put anywhere (e.g. airline), avoiding the pollution of the output buffer. If the user is interested in the timing she should not enqueue commands.